static/js/components/PrivacyTab.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+
+import ProfileTab from "../util/ProfileTab";
+import { saveAndContinue } from "../util/profile_edit";
+
+class PrivacyTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.saveAndContinue = saveAndContinue.bind(this, '/dashboard');
+  }
+
+  static propTypes = {
+    profile:        React.PropTypes.object,
+    errors:         React.PropTypes.object,
+    saveProfile:    React.PropTypes.func,
+    updateProfile:  React.PropTypes.func
+  };
+
+  static defaultProps = {
+    requiredFields: [
+      ['account_privacy']
+    ],
+    validationMessages: {
+      'account_privacy': 'Privacy'
```

Comment:
'Privacy' seems a little vague as a validation message to me, maybe something 'Privacy Level'  instead?